src/App.js

```diff
@@ -1,9 +1,12 @@
 import './App.css';
+import BookList from './containers/BookList';
+import BooksForm from './containers/ BooksForm';
 
 function App() {
   return (
     <div className="App">
-      Hello Mr Code Reviewer
+      <BookList />
+      <BooksForm />
     </div>
   );
```

Comment:
- [ ] App component should be placed at the `src/components` dir

Reply:
Done